### PR TITLE
Add FXIOS-8417 [v124] Firefox default browser onboarding footer string

### PR DESCRIPTION
**Pull request by nbhasin2**
## :scroll: Tickets
[Jira ticket - 8417](https://mozilla-hub.atlassian.net/browse/FXIOS-8417)
[Github issue - 18658](https://github.com/mozilla-mobile/firefox-ios/issues/18658)

## :bulb: Description
Added Firefox default browser onboarding footer string

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

**Review by dnarcese — APPROVED** [review]

File: firefox-ios/Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -1218,6 +1218,11 @@ extension String {
                 tableName: "Onboarding",
                 value: "3. Select *%@*",
                 comment: "The third label on the Default Browser Popup, which is a card with instructions telling the user how to set Firefox as their default browser. Placeholder is the app name. The *text inside asterisks* denotes part of the string to bold, please leave the text inside the '*' so that it is bolded correctly.")
+            public static let DescriptionFooter = MZLocalizedString(
+                key: "DefaultBrowserPopup.DescriptionFooter.v124",
+                tableName: "Onboarding",
+                value: "*Is %@ already your default?* Close this message and tap Skip.",
```
Comment (lmarceau) [inline firefox-ios/Client/Frontend/Strings.swift:1224]: We support bold through *bold* ?

Comment (nbhasin2) [reply firefox-ios/Client/Frontend/Strings.swift:1224]: we do, look above the file

Comment (lmarceau) [reply firefox-ios/Client/Frontend/Strings.swift:1224]: Yes I am asking, first time I see this format in a string 👀

Comment (lmarceau) [reply firefox-ios/Client/Frontend/Strings.swift:1224]: I guess it's only supported for Nimbus onboarding strings?

**Review by lmarceau — APPROVED** [review]

File: firefox-ios/Client/Frontend/Strings.swift (hunk 1 above)
Comment (nbhasin2) [reply firefox-ios/Client/Frontend/Strings.swift:1224]: Yeah, I saw it at couple of places. 

FYI @adudenamedruby feel free to update if this isn't the place for it as I saw you added others like this as well

**Pull request merged**